doc/containers.md

```diff
@@ -225,28 +225,30 @@ LXD supports different kind of network devices:
+nictype                 | string    | -                 | yes       | all                                       | -                                      | The device type, one of "bridged", "macvlan", "p2p", "physical", or "sriov"
+limits.ingress          | string    | -                 | no        | bridged, p2p                              | -                                      | I/O limit in bit/s for incoming traffic (supports kbit, Mbit, Gbit suffixes)
+limits.egress           | string    | -                 | no        | bridged, p2p                              | -                                      | I/O limit in bit/s for outgoing traffic (supports kbit, Mbit, Gbit suffixes)
+limits.max              | string    | -                 | no        | bridged, p2p                              | -                                      | Same as modifying both limits.ingress and limits.egress
+name                    | string    | kernel assigned   | no        | all                                       | -                                      | The name of the interface inside the container
+host\_name              | string    | randomly assigned | no        | bridged, macvlan, ipvlan, p2p, sriov      | -                                      | The name of the interface inside the host
+hwaddr                  | string    | randomly assigned | no        | all                                       | -                                      | The MAC address of the new interface
+mtu                     | integer   | parent MTU        | no        | all                                       | -                                      | The MTU of the new interface
+parent                  | string    | -                 | yes       | bridged, macvlan, ipvlan, physical, sriov | -                                      | The name of the host device or bridge
+vlan                    | integer   | -                 | no        | macvlan, physical                         | network\_vlan, network\_vlan\_physical | The VLAN ID to attach to
+ipv4.address            | string    | -                 | no*       | bridged, ipvlan                           | network                                | An IPv4 address to assign to the container through DHCP, for IPVLAN list of IPv4 for container (required at least one IPv4/IPv6)
+ipv6.address            | string    | -                 | no*       | bridged, ipvlan                           | network                                | An IPv6 address to assign to the container through DHCP, for IPVLAN list of IPv6 for container (required at least one IPv4/IPv6)
+nameserver              | string    | -                 | no        | ipvlan                                    | network                                | A list of Nameserver IPs that will be exposed to container exposed through devLXD
```

Comment:
This really isn't something that LXD should know or care about. It's fine for users to use a common `user` config key for it, but I don't think it deserves its own config key.

Reply:
ok, will remove nameserver key